pkg/kubelet/kubelet.go

```diff
@@ -1525,6 +1529,56 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 		metrics.DeprecatedPodStartLatency.Observe(metrics.SinceInMicroseconds(firstSeenTime))
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
+		//NOTE: Currently, CRI does not return CPU limit info with dockershim (docker API limitation)
+		//      If there's no existing API pod status upon create, then set pod status resources limits from pod spec.
+		//      If existing API pod status is found, then set limits from there if not reported by kube pod stattus
+		//TODO: DOES THIS HANDLE THE CASE WHERE CRI DOES NOT SUPPORT RETURNING CPU AND MEMORY LIMIT INFO?
```

Comment:
can we keep the comments to describe the intent of the code instead of asking question and listing TODOs? Maybe move TODOs to the tracking issue?

Reply:
i'll create a tracking issue for this. but this is on top of my mind since i need to discuss with sig-node - may need to update the CRI kep design based on their input.

Reply:
i'm going to update the code. i got a suggestion from the sig-node discussion this tuesday which simplifies the code. i tried a few different variations of QoS in pods and seems to work. this code is not necessary. i'll send an updated review by eod tomorrow